src/main/java/io/voucherify/client/module/CustomersModule.java

```diff
@@ -33,6 +36,13 @@ public CustomerResponse update(Customer customer) {
     return executeSyncApiCall(api.updateCustomer(customer.getId(), customer));
   }
 
+  public CustomerBulkUpdateResponse update(List<Customer> customers) {
+      if (customers.size() > 100){
```

Comment:
API will throw error `VoucherifyError(code=400, details=Specify at most 100 customers, key=too_many_customers)` which can be caught by `VoucherifyError` exception. We don't need to check this case in code. Limits also could be changed in future.